src/corona_hakab_model/main.py

```diff
@@ -79,6 +79,17 @@ def run_simulation(args):
         consts = Consts.from_file(args.simulation_parameters_path)
     else:
         consts = Consts()
+
+    # Sanity check for medical states time
+    time_parameters = ["latent_to_pre_symptomatic_days", "latent_to_asymptomatic_days",
```

Comment:
I don't like these hardcoded names. Maybe iterate over all the stochastic states in the machine?

Reply:
Maybe perform the check when a transfer is added to a stochastic state?

Reply:
Moved the check inside state_machine. This eliminates the ugly hardcoded names, but the check is performed later. In my opinion definitely with it